app/validators/date_validator.rb

```diff
@@ -15,6 +23,10 @@ def iso8601_regex
     /\A[0-9]{4}\-[0-9]{2}\-[0-9]{2}\z/
   end
 
+  def invalid_date_error_message
```

Comment:
We don't really need methods to return this string and the `error_message` string if these are only used once when added to the errors collection.